### ISSUE TITLE
Admin, Zones, Country filter does not handle multibyte characters/countries with an accent

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#4795]
ZC158
Add a test zone to Réunion.
Select RÉUNION from the dropdown list: no result.

**Comment by lat9** [comment]
A bit more information is needed.  Which admin page displays this issue?

**Comment by torvista** [comment]
Sorry,
Admin->Locations/Taxes->Zones->New Zone

**Comment by lat9** [comment]
OK, I went to that admin page and added a `Réunion` zone, selecting "SPAIN" as the associated country.  That displayed subsequently as one of the zones for Spain.  What else is needed to reproduce the issue?

**Comment by torvista** [comment]
No, add a TestZone to the country Reunion. Then Select Reunion from the dropdown: no result/zone.
